app/uploaders/image_uploader.rb

```diff
@@ -36,9 +35,13 @@ def store_dir
   # end
 
   # Create different versions of your uploaded files:
-  # version :thumb do
-  #   process resize_to_fit: [50, 50]
-  # end
+  version :thumb do
+    process resize_to_fill: [400, 400]
+  end
+
+  version :big do
+    process resize_to_fill: [1000, 1000]
+  end
```

Comment:
you can leave big image as original without resizing, so this version is redundant,  thumb is ok

Reply:
Yes, but use the image of the original version, you may be confused by this
![image](https://user-images.githubusercontent.com/23640391/156742597-477f5140-2f5b-4fbd-a33d-c85f402cb4aa.png)